src/runtime/os_wasip1.go

```diff
@@ -45,8 +45,15 @@ type iovec struct {
 	bufLen size
 }
 
+//go:wasmimport env require
+func require(code int32)
+
 func exit(code int32) {
-	return
+	if code == 0 {
+		wasmPause()
```

Comment:
Should we add a TODO or note here?  As unlike require(0), wasmPause() will not exit immediately - it will wait until goroutine scheduler exits in proc.go.

Further, it looks like we can remove all wasmPause() definition in other GOOS's if wasmPause() is only called in GOOS=wasip1 (os_wasip1.go) file.

Reply:
Yep you are right.

Reply:
As for removing all wasmPause() in other lock_{PLATFORM}.go, doing this will lead to go build failing. It seems failing in cross-compilation stage. And to keep code simple, we would still remain them as they were.